Subject: mac80211: remove wiphy_to_hw

This isn't used by anyone, if we ever need it we can add
it back, until then it's useless.

Signed-off-by: Johannes Berg <johannes@sipsolutions.net>
---
 include/net/mac80211.h |    2 --
 net/mac80211/cfg.c     |    7 -------
 2 files changed, 9 deletions(-)

--- everything.orig/include/net/mac80211.h	2008-10-07 20:05:49.000000000 +0200
+++ everything/include/net/mac80211.h	2008-10-07 20:06:42.000000000 +0200
@@ -867,8 +867,6 @@ struct ieee80211_hw {
 	u8 max_altrate_tries;
 };
 
-struct ieee80211_hw *wiphy_to_hw(struct wiphy *wiphy);
-
 /**
  * SET_IEEE80211_DEV - set device for 802.11 hardware
  *
--- everything.orig/net/mac80211/cfg.c	2008-10-07 20:06:37.000000000 +0200
+++ everything/net/mac80211/cfg.c	2008-10-07 20:06:42.000000000 +0200
@@ -17,13 +17,6 @@
 #include "rate.h"
 #include "mesh.h"
 
-struct ieee80211_hw *wiphy_to_hw(struct wiphy *wiphy)
-{
-	struct ieee80211_local *local = wiphy_priv(wiphy);
-	return &local->hw;
-}
-EXPORT_SYMBOL(wiphy_to_hw);
-
 static bool nl80211_type_check(enum nl80211_iftype type)
 {
 	switch (type) {
